www/style.css

```diff
@@ -22,3 +27,17 @@ sub {
   font-style: italic;
   font-size: 20px;
 }
+
+.labelIncorrect {
+  border: 2px solid rgba(255, 0, 0, 0.5);
+  padding: 5px;
+  border-radius: 5px;
+  display: inline-block;
+}
+
+.labelCorrect {
+  border: 2px solid rgba(0, 255, 0, 0.5);
+  padding: 5px;
+  border-radius: 5px;
+  display: inline-block;
+}
```

Comment:
It might make sense to use only one class here to avoid style duplication. The CSS could be something like this:

```css
#check-code-result {
  border: 2px solid rgba(255, 0, 0, 0.5);
  padding: 5px;
  border-radius: 5px;
  display: inline-block;
}

.success {
    border: 2px solid rgba(0, 255, 0, 0.5);
}
```